widgetssdk/src/main/java/com/glia/widgets/core/engagement/domain/IsCallVisualizerUseCase.kt

```diff
@@ -0,0 +1,14 @@
+package com.glia.widgets.core.engagement.domain
+
+import com.glia.androidsdk.Engagement
+import com.glia.androidsdk.Glia
+import com.glia.androidsdk.omnibrowse.OmnibrowseEngagement
+
+
+class IsCallVisualizerUseCase {
+    fun execute(): Boolean {
```

Comment:
Is it possible to use invoke() here?

Reply:
I don't think so as far as I understand because they aren't used immediately.

Reply:
Sorry, haven't got your point. I still think it's possible to replace it with `invoke()` here as we do for other use cases now. If it's called from Java, then it's still possible to call `.invoke()`. For Kotlin it will be without `.invoke()`.

Anyway, it's not obligatory to use invoke. Just wanted to say that it should be possible to do it here.